api/src/test/java/org/openmrs/module/bedmanagement/BedManagementServiceImplTest.java

```diff
@@ -150,6 +152,54 @@ public void shouldGetBedDetailsWithMultiplePatientInformationsById() {
 		assertEquals("M", actualPatient2.getGender());
 	}
 	
+	@Test
+	public void shouldReturnCurrentBedAssignmentWhenVoidedIsFalse() {
```

Comment:
Since this ticket is about queries for data that is already in the database, i find it better to simply include some voided rows in these dataset files and then ensure that they are excluded from the returned results: https://github.com/openmrs/openmrs-module-bedmanagement/tree/master/api/src/test/resources